### ISSUE TITLE
_get_login_path_file uses getenv() instead of os.getenv() (typo?)

### ISSUE THREAD

**Issue by tybin** [PyMySQL/myloginpath#5]
Problem:
```
  File "backup.py", line 20, in __init__
    self.path = myloginpath.parse(path)
  File "C:\Users\user\project\.env\lib\site-packages\myloginpath.py", line 35, in parse
    path = _get_login_path_file()
  File "C:\Users\user\project\.env\lib\site-packages\myloginpath.py", line 53, in _get_login_path_file
    file_path = os.path.join(getenv("APPDATA"), "MySQL", ".mylogin.cnf")
NameError: name 'getenv' is not defined

```
Fix:
```
52 if sys.platform == "win32":
53        file_path = os.path.join(os.getenv("APPDATA"), "MySQL", ".mylogin.cnf")
```

**Comment by tybin** [comment]
myloginpath==0.0.1

**Comment by methane** [comment]
released 0.0.2
